tests/testdata/demo-modules/images/demo-module-3/can-load.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Can load function to determine if module already marge in WordPress core.
+ *
+ * @since   n.e.x.t
+ * @package performance-lab
+ */
+
+return function() {
```

Comment:
You can replace this with:

```suggestion
return __return_true;
```

Reply:
Sorry, I missed it completely. Thanks for pointing it out.